StoryMap/StoryMap/Screens/StoryDetail/View/StoryDetailViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  StoryDetailViewController.swift
+//  StoryMap
+//
+//  Created by Dory on 02/11/2021.
+//
+
+import Foundation
+import UIKit
+
+class StoryDetailViewController: UIViewController {
+    private var viewModel: StoryDetailViewModelType
+
+    private let imageView = UIImageView()
+    private var tagField = UITextField()
+    
+    init(viewModel: StoryDetailViewModelType) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        title = viewModel.story.title
+        view.backgroundColor = .white
+        setupImageView()
+//        setupMapView()
+
+//        if owner = "me" { setupTagLabel() } else
+        setupTagTextField()
+//        setupPrivateToggle()
+//        setupAddStoryBook()
+//        How to change title if title is in NavigationBar?
```

Comment:
`title = "My title"` changes the title in NavigationBar

Reply:
`navigationItem.title`

Reply:
ok